mocks_test.go

```diff
@@ -65,3 +66,13 @@ func (m *mockDeviceAccess) authorizeWRP(ctx context.Context, message *wrp.Messag
 	arguments := m.Called(ctx, message)
 	return arguments.Error(0)
 }
+
+type mockJWTParser struct {
+	mock.Mock
+}
+
+func (parser *mockJWTParser) ParseJWT(token string, claims jwt.Claims, parseFunc jwt.Keyfunc) (*jwt.Token, error) {
+	arguments := parser.Called(token)
```

Comment:
all parameters should be passed in mock struct
```suggestion
	arguments := parser.Called(token, claims, parseFunc)
```